tests/rptest/tests/rpk_group_test.py

```diff
@@ -0,0 +1,151 @@
+
+        self.rpk.group_delete(group_1)
+        groups_2 = self.rpk.group_list()
+        assert group_1 not in groups_2
+
+    @cluster(num_nodes=5)
+    def test_group_describe(self):
+        """
+        Check for group describe when the group is stable.
+        """
+        topic = "test_group"
+        self.setup_producer(3, topic)
+
+        group_1 = "test-g1"
+        consumer = RpkConsumer(self._ctx, self.redpanda, topic, group=group_1)
+        consumer.start()
+
+        rpk = RpkTool(self.redpanda)
+
+        wait_until(lambda: rpk.group_describe(group_1).state == "Stable",
+                   timeout_sec=30,
+                   backoff_sec=1)
+
+        self.producer.stop()
+        consumer.stop()
+
+    @cluster(num_nodes=1)
+    def test_empty_group_describe(self):
+        """
+        Check for group describe when the group is empty.
+        Reproduce: https://github.com/redpanda-data/redpanda/commit/ab6a99162056e2980ebf873ed7192878419769ae
+        """
+        topic = "test_group"
+        self._rpk_produce_to_topic(topic, 20)
+
+        group_1 = "test-g1"
+        self.rpk.consume(topic, group=group_1, n=10)
+
+        rpk_group = self.rpk.group_describe(group_1)
+
+        assert rpk_group.state == "Empty"
+        assert rpk_group.name == group_1
+        assert rpk_group.partitions[0].current_offset == 10
+        assert rpk_group.partitions[0].lag == 10
+
+    @cluster(num_nodes=1)
+    def test_group_seek_to(self):
+        """
+        Check rpk group seek `--to end` and `--to start`
+        """
+        topic = "test_group"
+        self._rpk_produce_to_topic(topic, 20)
+
+        group_1 = "test-g1"
+        self.rpk.consume(topic, group=group_1, n=10)
+
+        rpk_group = self.rpk.group_describe(group_1)
+        assert rpk_group.partitions[0].current_offset == 10
+
+        self.rpk.group_seek_to(group_1, "end")
+        rpk_group = self.rpk.group_describe(group_1)
+        assert rpk_group.partitions[0].current_offset == 20
+
+        self.rpk.group_seek_to(group_1, "start")
+        rpk_group = self.rpk.group_describe(group_1)
+        assert rpk_group.partitions[0].current_offset == 0
+
+    @cluster(num_nodes=1)
+    def test_group_seek_to_group(self):
+        """
+        Check merge operation of `rpk group seek --to-group`
+        """
+        topic = "test_group"
+        self._rpk_produce_to_topic(topic, 30)
+
+        group_1 = "test-g1"
+        self.rpk.consume(topic, group=group_1, n=10)
+
+        group_2 = "test-g2"
+        self.rpk.consume(topic, group=group_2, n=20)
+
+        rpk_group_1 = self.rpk.group_describe(group_1)
+        rpk_group_2 = self.rpk.group_describe(group_2)
+
+        assert rpk_group_1.partitions != rpk_group_2.partitions
+
+        self.rpk.group_seek_to_group(group_1, group_2)
+        rpk_group_1 = self.rpk.group_describe(group_1)
+        rpk_group_2 = self.rpk.group_describe(group_2)
+
+        assert rpk_group_1.partitions == rpk_group_2.partitions
```

Comment:
Can this test add a `--to-file` test?

Reply:
Also can this add a timestamp based seek test?
* grab time before test
* produce 5 messages
* sleep 1s, grab current time
* produce 5 more messages
* sleep 1s, grab final time
* seek to time before test, ensure we can consume 10
* seek to middle time, ensure we can consume only the last 5
* seek to final time, ensure we are at end